sortedm2m/static/sortedm2m/widget.js

```diff
@@ -19,18 +15,31 @@ if (typeof jQuery === 'undefined') {
                 var label, labelFor;
                 var currentElement = ul;
 
-                while (!label || !label.length) {
+                while (true) {
                     currentElement = currentElement.parent();
+                    if(currentElement[0].tagName == "BODY")
+                        // we can not guess the label.
+                        break;
                     label = currentElement.siblings('label');
```

Comment:
This will break I think if there is multiple `currentElement.siblings` return several nodes.

Reply:
Actually no.